lib/LaTeXML/Util/Pathname.pm

```diff
@@ -406,6 +406,8 @@ sub pathname_kpsewhich {
     close($resfh);             # ignore exit status (only one of @candidates exists, usually)
     if ($result) {
       chomp $result;
+      my ($vol, $dir, $name) = File::Spec->splitpath($result);
+      LaTeXML::Common::Error::Warn('unexpected', "$name", 'pathname_kpsewhich', "not among the requested files (@candidates)") if !grep { $_ eq $name } @candidates;
```

Comment:
I am not sure why `use LaTeXML::Common::Error` does not work, by the way. I don't always understand perl.

Reply:
> Presumably there will end up some error or warning higher up that the file couldn't be found?

I think the answer must be yes, or I wouldn't have found the source, but it was hard to debug. You only get an error about a file not existing and no explanation. Very puzzling when it happens.